Update App.vue

nickdex opened the pull request:
Fix empty response to the search annotation request

Resolves #2

nickdex committed:
Update App.vue

Fix empty response to the search annotation request

c6p commented:
@nickdex thanks for the PR. But hypothes.is [is not the only authority](https://web.hypothes.is/help/overview-of-the-hypothesis-system/), hence `c6p@hypothes.is` as an account. However, since 99% of the users will just use hypothes.is, I'm all for defaulting to it.

Could you modify your PR, so when user account does not contain `@`, it adds `@hypothes.is` else leaves it unmodifed?

nickdex committed:
Update App.vue

Per https://github.com/c6p/logseq-hypothesis/pull/3#issuecomment-932397903, add hypothes.is as default authority and allow user to add others if needed

nickdex commented:
@c6p Ah okay, I was using it wrong then! 
Like you said, hypothes.is will be what most people so update the PR accordingly.

P.S. - Would you mind adding ['hacktoberfest-accepted' label](https://hacktoberfest.digitalocean.com/resources/participation) to this PR?

c6p commented on src/App.vue:
```diff
@@ -107,7 +107,7 @@ export default {
           limit: 200,
           order: 'asc',
-          user: `acct:${this.user}@hypothes.is`,
+          user: `acct:${this.user}${this.user.includes('@') ? '@hypothes.is' : '' }`,
```
Ternary reversed, could you also fix that? If not has `@` add hypothes.is. Then all is fine.

nickdex replied:
Whoops! fixed it.

nickdex committed:
Update App.vue

Fixed the logic for adding hypothes.is as default authority

c6p reviewed (approved):
Thanks for the PR.
